woodwork/tests/utils/test_utils.py

```diff
@@ -243,87 +247,107 @@ def test_is_s3():
-
-
-def test_reformat_to_latlong_list():
-    simple_latlong = [1, 2]
-
-    assert _reformat_to_latlong((1, 2), use_list=True) == simple_latlong
-    assert _reformat_to_latlong(("1", "2"), use_list=True) == simple_latlong
-    assert _reformat_to_latlong("(1,2)", use_list=True) == simple_latlong
-
-    assert _reformat_to_latlong([1, 2], use_list=True) == simple_latlong
-    assert _reformat_to_latlong(["1", "2"], use_list=True) == simple_latlong
-    assert _reformat_to_latlong("[1, 2]", use_list=True) == simple_latlong
-    assert _reformat_to_latlong("1, 2", use_list=True) == simple_latlong
-
-    assert _reformat_to_latlong((1, np.nan), use_list=True) == [1, np.nan]
-    assert _reformat_to_latlong((np.nan, "1"), use_list=True) == [np.nan, 1]
-
-    # This is how csv and parquet will deserialize
-    assert _reformat_to_latlong("[1, nan]", use_list=True) == [1, np.nan]
-    assert _reformat_to_latlong("[1, NaN]", use_list=True) == [1, np.nan]
-
-
-def test_to_latlong_float():
-    assert _to_latlong_float(4) == 4.0
-    assert _to_latlong_float("2.2") == 2.2
-
-    assert _to_latlong_float(None) is np.nan
-    assert _to_latlong_float(np.nan) is np.nan
-    assert _to_latlong_float(pd.NA) is np.nan
-
-    error = re.escape(
-        "Latitude and Longitude values must be in decimal degrees. The latitude or longitude represented by [1, 2, 3] cannot be converted to a float."
-    )
-    with pytest.raises(ValueError, match=error):
-        _to_latlong_float([1, 2, 3])
+@pytest.mark.parametrize(
+    "test_input, error",
+    [
+        ({1, 2, 3}, LatLongIsNotTupleError),
+        ("{1, 2, 3}", LatLongIsNotTupleError),
+        ("This is text", LatLongIsNotTupleError),
+        ("'(1,2)'", LatLongIsNotTupleError),
+        ((1, 2, 3), LatLongLengthTwoError),
+        ("(1, 2, 3)", LatLongLengthTwoError),
+        ("(1,)", LatLongLengthTwoError),
+        (("41deg52'54\" N", "21deg22'54\" W"), LatLongIsNotDecimalError),
+    ],
+)
+def test_reformat_to_latlong_errors(test_input, error):
+    with pytest.raises(error):
+        _reformat_to_latlong(test_input)
+
+
+@pytest.mark.parametrize(
+    "test_input,expected",
+    [
+        ((1, 2), (1, 2)),
+        (("1", "2"), (1, 2)),
+        ("(1,2)", (1, 2)),
+        ([1, 2], (1, 2)),
+        (["1", "2"], (1, 2)),
+        ("[1, 2]", (1, 2)),
+        ("1, 2", (1, 2)),
+        ((1, np.nan), (1, np.nan)),
+        ((np.nan, "1"), (np.nan, 1)),
+        ("(1, nan)", (1, np.nan)),
+        ("(NaN, 9)", (np.nan, 9)),
+        ("(1, None)", (1, np.nan)),
+        ("(<NA>, 9)", (np.nan, 9)),
+        ((np.nan, np.nan), (np.nan, np.nan)),
+        ((pd.NA, pd.NA), (np.nan, np.nan)),
+        ((None, None), (np.nan, np.nan)),
+        (None, np.nan),
+        (np.nan, np.nan),
+        (pd.NA, np.nan),
+        ("None", np.nan),
+        ("NaN", np.nan),
+        ("<NA>", np.nan),
+    ],
+)
+def test_reformat_to_latlong(test_input, expected):
```

Comment:
if the only difference between `test_reformat_to_latlong` and `test_reformat_to_latlong_koalas`, could parametrize `is_koalas` True/False and cast from list to tuple or vice versa if expected needs to be a list and is a tuple